src/clients/inst_crowbar.rb

```diff
@@ -47,15 +48,20 @@ def main
 
       @dialog_ret = :auto
 
-      Wizard.CreateDialog if Mode.normal
+      if Package.Installed("crowbar-core")
 
-      Progress.off
-      @dialog_ret = CrowbarSequence()
-      Progress.on
+        Wizard.CreateDialog if Mode.normal
 
-      Wizard.CloseDialog if Mode.normal
+        Progress.off
+        @dialog_ret = CrowbarSequence()
+        Progress.on
 
-      deep_copy(@dialog_ret)
+        Wizard.CloseDialog if Mode.normal
+      else
+        Builtins.y2milestone("Necessary packages not installed, skipping Crowbar configuration...")
+      end
+
+      @dialog_ret
```

Comment:
No, that should not be a problem, the return value is a `Symbol` which is created here and symbols are singletons anyway....